src/main/java/org/eolang/ineo/FuseMojo.java

```diff
@@ -100,17 +107,17 @@ public void execute() {
                                         .set(pckg)
                                 ).xml()
                             ),
-                            this.sourcesDir.toPath().resolve(
-                                String.join(File.separator, pckg, "BA.xmir")
-                            )
+                            generated
                         )
                     ).save();
+                    Logger.info(this, "New XMIR was generated: %s", generated);
                 } catch (final IOException ex) {
                     throw new IllegalStateException("Couldn't save file after transformation", ex);
                 } catch (final ImpossibleModificationException ex) {
                     throw new IllegalStateException("Couldn't transform fused XMIR", ex);
                 }
             }
         }
+        Logger.info(this, "Fused %d files", 1);
```

Comment:
@maxonfjvipon in general, it's better to log this way: https://github.com/objectionary/eo/blob/master/eo-maven-plugin/src/main/java/org/eolang/maven/OptimizeMojo.java#L107-L115